src/docs/product/relay/options.mdx

```diff
@@ -126,3 +126,9 @@ default.
 | -------------- | ------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | sentry.enabled | Boolean | default: `false` | Determines whether to report internal errors to a separate DSN. `false` means that no internal errors will be sent, but will still be logged.                                        |
 | sentry.dsn     | String  | optional: `true` | Sentry DSN to which you can report internal Relay failures. We recommend setting this to a value that sends Relay errors directly to Sentry instead of sending them to itself first. |
+
+## Geoip
+
```

Comment:
By default, Relay does not include geographical location information in events. Set the MaxMind database file path to add GeoIP info in events.

Reply:
I think it'd be helpful to add an intro sentence for this section, something like above.